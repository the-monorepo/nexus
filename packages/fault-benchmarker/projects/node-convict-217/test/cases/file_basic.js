'use strict';

exports.conf = {
  ip: {
    doc: 'The IP Address to bind.',
    default: '127.0.0.1',
    format: 'ipaddress',
    env: 'IP_ADDRESS'
  },
  port: {
    default: 0,
    format: 'port',
    env: 'PORT',
    doc: 'The port to bind.'
  },
  session: {
    doc: 'Duration of the session',
    format: 'duration',
    default: '2 days'
  },
  cache: {
    doc: 'Duration of the cache',
    format: 'duration',
    default: '1 hour'
  }
};
