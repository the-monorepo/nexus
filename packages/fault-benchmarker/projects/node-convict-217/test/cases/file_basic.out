{
  "ip": "10.0.1.101",
  "port": 8080,
  "session": 345600000,
  "cache": 3000
}
